package.json

```diff
@@ -201,11 +202,10 @@
     "stylelint-config-standard": "^18.2.0",
     "uglifyjs-webpack-plugin": "1.2.7",
     "url-loader": "^1.0.1",
-    "webpack": "^4.16.3",
-    "webpack-bundle-analyzer": "^2.13.1",
-    "webpack-cli": "^3.1.0",
-    "webpack-dev-server": "^3.1.5",
-    "webpack-merge": "^4.1.3",
+    "webpack": "^3.10.0",
```

Comment:
Had to downgrade, eh? Why is that?

Reply:
Yes! I had to downgrade because because I needed to install 'extract-text-webpack-plugin' to get this to work, and it requires an older version of webpack. Things broke when I tried using the most recent version. :-/